.travis.yml

```diff
@@ -1,20 +1,23 @@
-dist: xenial
+dist: focal
 language: c
-os: linux
 
 env:
   global:
-    - LLVM_VERSION=3.9
-    - GCC_VERSION=4.9
+    - LLVM_VERSION=7
+    - GCC_VERSION=7
     - GCC=gcc-${GCC_VERSION}
     - GXX=g++-${GCC_VERSION}
+    - GFORTRAN=gfortran-${GCC_VERSION}
+    - CLANG=clang-${LLVM_VERSION}
+    - FLANG=flang-${LLVM_VERSION}
 
 addons:
   apt:
     update: true
     packages:
       - libmpfr-dev
-      - gcc-4.9 gcc-4.9-plugin-dev g++-4.9 gfortran-4.9 libgfortran-4.9-dev
+      - gcc-7 g++-7 gfortran-7 libgfortran-7-dev
```

Comment:
Could we remove gfortran and libgfortran dependency now ?

Reply:
Can we also compile verificarlo with llvm instead of using gcc and g++ ? That would simplify a lot the Docker image.

Reply:
> Could we remove gfortran and libgfortran dependency now ?

Yes we can.

> Can we also compile verificarlo with llvm instead of using gcc and g++ ? That would simplify a lot the Docker image.

Unfortunatly not because float128 type is not supported by clang yet.

Reply:
float128 provided with gcc/g++ is merely 5-6 times slower than mpfr when targeting this accuracy.  Is the additional complexity worth the price ?

Reply:
I think it's really worth keeping, since float128 will be soon provided in hardware anyways. And the quad backend is a thing of beauty, we don't want to scrap it ;-)